tests/Nox.Types.Tests/Types/HashedText/HashedTextTests.cs

```diff
@@ -1,11 +1,79 @@
 ﻿// ReSharper disable once CheckNamespace
+using System.Security.Cryptography;
+using FluentAssertions;
+
 namespace Nox.Types.Tests.Types;
 
 public class HashedTextTests
 {
+
+    [Fact]
+    public void HashedText_Constructor_WithoutOptions_ReturnsHashedValue()
+    {
+        string text = "Text to hash";
+        var hashedText = HashedText.From(text);
+
+        hashedText.Should().NotBeNull();
+        hashedText.Value.Salt.Should().NotBeNull().And.NotBe(text);
+    }
+
+
+    [Fact]
+    public void HashedText_Constructor_WithOptions_ReturnsHashedValue()
+    {
+        string text = "Text to hash";
+        byte[] textData = System.Text.Encoding.UTF8.GetBytes(text);
+        byte[] hash = SHA512.HashData(textData);
+        var textHashedExpected = Convert.ToBase64String(hash);
+
+        var hashedText = HashedText.From(text, new HashedTextTypeOptions() { HashingAlgorithm = HashingAlgorithm.SHA512, Salt = 0 });
```

Comment:
Just for readability can you move this line under "string text = "Text to hash""?
Same for the other methods below.